app/controllers/users_controller.rb

```diff
@@ -18,6 +26,26 @@ def create
     end
   end
 
+  def edit; end
+
+  def update
+    if @user.update_attributes(user_params)
+      flash[:success] = t "users.users_controller.profile_updated"
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t "users.users_controller.user_deleted"
+      redirect_to users_path
+    else
+      flash[:warning] = t "users.users_controller.failed"
```

Comment:
Nếu k thành công thì nó nhảy về đâu em

Reply:
ko thành công thì mình in ra flash rồi  cho hắn về lại trang chủ lun hà anh?